pipelines/pyproject.toml

```diff
@@ -15,17 +15,17 @@ packages = [
 [tool.poetry.dependencies]
 python = ">=3.9,<3.11"
 Jinja2 = ">=3.0.1,<4.0.0"
-google-cloud-aiplatform = ">=1.30.1"
-google-cloud-storage = "<2.13.0"
-google-cloud-pipeline-components = "^2.1.0"
+google-cloud-aiplatform = ">=1.54.1"
+google-cloud-storage = "<2.17.0"
```

Comment:
@theo-dt there has been a bug which surfaced with a warning when triggering a pipeline job. so take care that this version is correct.

Reply:
Where did you see that warning because I cant see a warning anywhere